tests/test_slots.py

```diff
@@ -298,3 +316,52 @@ class C2(C1Bare):
     hash(c2)  # Just to assert it doesn't raise.
 
     assert {"x": 1, "y": 2, "z": "test"} == attr.asdict(c2)
+
+
+@pytest.mark.skipif(PY2, reason="closure cell rewriting is PY3-only.")
+def test_closure_cell_rewriting():
```

Comment:
Hm is this a pytest feature I don't completely understand? I usually use parametrize to provide multiple values of a fixture.